src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(final List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> filterWinners() {
+        return cars.stream()
+            .filter(car -> car.distance() == calculateMax())
+            .collect(Collectors.toList());
+    }
```

Comment:
`maxPosition`을 변수에 담아 처리하지 않으면, 매 루프마다 `calculateMax()` 불필요한 연산을 수행하게 됩니다.

```suggestion
        int maxPosition = calculateMax()
        return cars.stream()
            .filter(car -> car.distance() == maxPosition)
            .collect(Collectors.toList());
    }
```

Reply:
비효율적으로 `calculateMax()`가 계속 반복 실행되었었네요! 변수로 따로 빼도록 하겠습니다.